[DOC-750] Voyager v1.8.17 docs changes (#27046)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/reference/schema-migration/import-schema.md

        --post-snapshot-import true \
        --refresh-mviews true
```

> {{< note title ="Note" >}}

Reviewer: ```suggestion
{{< note title ="Deprecated flags" >}}
```

> {{< note title ="Note" >}}

Reviewer: I would put this note in the front section above Syntax

> The --post-import-data and --refresh-mviews flags are now deprecated in the import schema command. In their place, a new command has been introduced: [finalize-schema-post-data-import](../finalize-schema-post-data-import/)

Reviewer 2: Put these flags under code in a way similar to other notes.

> The `--post-import-data` and `--refresh-mviews` flags are now deprecated in the import schema command. In their place, a new command has been introduced: [finalize-schema-post-data-import](../finalize-schema-post-data-import/)

Reviewer: ```suggestion
The `--post-import-data` and `--refresh-mviews` flags are deprecated. Use [finalize-schema-post-data-import](../finalize-schema-post-data-import/) instead.
```
